produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
+
+      if merchant_exists? merchant_identifier
+        UI.success("[DevCenter] Merchant '#{merchant_identifier})' already exists, nothing to do on the Dev Center")
+      else
+        merchant_name = Produce.config[:merchant_name] || merchant_identifier.split(".").map(&:capitalize).reverse.join(' ')
```

Comment:
Ah, thanks for citing that other file with the same pattern, yeah I think if we could break it out into more digestible lines that would be much better, we should probably also do the same in the file you referenced too 😄